ci/test-config.yml

```diff
@@ -164,3 +164,12 @@ persistence:
       hostPath: "/tmp/outputs"
 
 environment: testing
+
+privateRegistries:
+  - host: "my.registry.com"
+    username: "_json_key"
```

Comment:
Here as well `user` instead of `username`. 

I would add an example of uri spec for the e2e test:

```
  - "user-foo:tKV9DYsxrwfiY85KYpf9N4GZ@cloud.somewhere.io:5000"
```